samples/notifications/ToastNotificationSample/ToastNotificationSample/ViewModels/ActivatedFromToastViewModel.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+
+using ToastNotificationSample.Helpers;
+using Windows.ApplicationModel.Activation;
+
+namespace ToastNotificationSample.ViewModels
+{
+    public class ActivatedFromToastViewModel : Observable
+    {
+        private string _activationSource;
+
+        public string ActivationSource
+        {
+            get => _activationSource;
```

Comment:
code in the sample should be in the same style as in the generates code. DOn't use lambdas where the generated code wouldn't